src/libs/elektra/plugin.c

```diff
@@ -96,7 +96,7 @@ int elektraProcessPlugin (Key * cur, int * pluginNumber, char ** pluginName, cha
 			referenceNameSize = fullsize - pluginNameSize - 4;
 			++iter; /* advance to one after hash */
 			*referenceName = elektraMalloc (referenceNameSize + sizeof (prefixReferenceName));
-			strncpy (*referenceName, prefixReferenceName, sizeof (prefixReferenceName));
+			strcpy (*referenceName, prefixReferenceName);
```

Comment:
Is there any advantage for this change? I think this will give a warning on some compilers (openBSD iirc)